hybris/bin/custom/tatacliq/tatacliqcore/resources/tatacliqcore-items.xml

```diff
@@ -146,6 +146,22 @@
 					</attribute>
 				</attributes>
 			</itemtype>
+			<!--Start of  Story ID- 105 -->
+			<itemtype code="CustomComponent" extends="SimpleCMSComponent" generate="true" autocreate="true"
```

Comment:
Rename this as AirpointsComponent, Follow naming convention and coding practices